src/fides/api/util/tcf/tcf_experience_contents.py

```diff
@@ -201,7 +268,12 @@ def get_matching_privacy_declarations(db: Session) -> Query:
         matching_privacy_declarations = matching_privacy_declarations.filter(
             NOT_AC_SYSTEM_FILTER
         )
-    return matching_privacy_declarations
+
+    return (
+        matching_privacy_declarations,
+        consent_legal_basis_filter,
+        legitimate_interest_legal_basis_filter,
```

Comment:
These filters need to be passed forward to further filter the various sections of the TCF Privacy Experience on legal basis.  We can't generate them after the fact because they are filters on the fields in the subquery.  They are later chained onto the base "matching_privacy_declarations" filter. in plus.